framework/source/class/qx/ui/list/List.js

```diff
@@ -597,6 +625,28 @@ qx.Class.define("qx.ui.list.List",
     },
 
 
+    /**
+     * Event handler for the updated event of the 
+     * qx.ui.virtual.layer.WidgetCell layer.
+     *
+     * Recalculates the item sizes in a deffered call,
+     * which only happens if we have variable item heights
+     */
+    _onLayerUpdated: function () {
+      if (this.isVariableItemHeight() === false) {
+        return;
+      }
+
+      if (this.__deferredLayerUpdate === null) {
+        this.__deferredLayerUpdate = new qx.util.DeferredCall(function () {
+          this._setRowItemSize();
+        }, this);
+      }
+
+      this.__deferredLayerUpdate.schedule();
+    },
+
```

Comment:
no, only the once.  `DeferredCall.schedule` puts the call on a queue, but the queue is not serviced until the event loop runs.  If you call `.schedule` several times, then nothing happens if it's already on the queue so it will only get called once when the event loop is freed up